### PR TITLE
chore: return migrator address on ReadImplementationAddresses output

### PR DESCRIPTION
**Pull request by 0xiamflux**
closes OPT-1410

### CODE REVIEW

File: packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol [hunk 1]
```diff
@@ -0,0 +1,375 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```
Comment (0xniha) [inline packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol]: We should set the version to a fixed one.
```suggestion
pragma solidity 0.8.15;
```

Comment (0xniha) [reply packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol]: have you ran just pr?

File: packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol
Comment (0xniha) [inline packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol:1]: Consider adding a test case for reverting when opcm code length is zero.

**Review by 0xniha — CHANGES_REQUESTED** [review]

File: packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol (hunk 1 above)
Comment (0xiamflux) [reply packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol]: Addressed [here](https://github.com/defi-wonderland/optimism/pull/819/commits/b143fc8731e349b2fb50f0830e70f70aa10a2a79) and [here](https://github.com/defi-wonderland/optimism/pull/819/commits/ffa9c6153932ab9d1f6c11d5910e661732adb105)

File: packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol
Comment (0xiamflux) [reply packages/contracts-bedrock/test/scripts/ReadImplementationAddresses.t.sol:1]: Addressed [here](https://github.com/defi-wonderland/optimism/pull/819/commits/6f3c5cc6e747339889a41fb682f5fc8eb43d1e92)

**Review by 0xniha — APPROVED** [review]
